Packs/Base/TestPlaybooks/playbook-Domain_extraction_test.yml

```
@@ -729,7 +730,48 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 1985,
+          "x": 1770,
+          "y": 1390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "87":
+    id: "87"
+    taskid: dd18176b-6915-4661-808b-748d3cf3483f
+    type: condition
+    task:
+      id: dd18176b-6915-4661-808b-748d3cf3483f
+      version: -1
+      name: non extraction - `www.walla.co.il`
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "77"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: ${Domain.Name}
+            iscontext: true
+          right:
+            value:
+              simple: '`www.walla.co.il`'
```

```suggestion
              simple: 'www.walla.co.il`'
```
It wont capture the first tick.